AABB_tree/doc/AABB_tree/Concepts/AABBGeomTraits_2.h

```diff
@@ -0,0 +1,202 @@
+
+/*!
+\ingroup PkgAABBTreeConcepts
+\cgalConcept
+
+The concept `AABBGeomTraits_2` defines the requirements for the first template parameter of the class
+`CGAL::AABB_traits_2<AABBGeomTraits_2, AABBPrimitive>`. It provides predicates and constructors to detect
+and compute intersections between query objects and the primitives stored in the AABB tree.
+In addition, it contains predicates and constructors to compute distances between a point query
+and the primitives stored in the AABB tree.
+
+\cgalRefines{SearchGeomTraits_2}
+
+\cgalHasModelsBegin
+\cgalHasModelsBare{All models of the concept `Kernel`}
+\cgalHasModelsEnd
+
+\sa `CGAL::AABB_traits_2<AABBGeomTraits_2,AABBPrimitive>`
+\sa `CGAL::AABB_tree<AABBTraits>`
+\sa `AABBPrimitive`
+
+*/
+
+class AABBGeomTraits_2 {
+public:
+
+/// \name Types
+/// @{
+
+/*!
+A functor object to detect intersections between two geometric objects.
+Provides the following operators:
+
+`bool operator()(Query, Bbox_2)`,
+
+`bool operator()(Query, Primitive::Datum)`,
+
+`bool operator()(Circle_2, Bbox_2)`.
+
+The operator returns `true` iff there exists a non-empty intersection.
+*/
+typedef unspecified_type Do_intersect_2;
+
+/*!
+A functor object to construct the intersection between two geometric objects.
+
+Provides the operator:
+
+`return_type operator()(const Query& q, const Primitive::Datum& d)`,
+
+which computes the intersection between `q` and `d`. The type of the returned object
+must be a `std::optional` of a `std::variant` of the possible intersection types.
+*/
+typedef unspecified_type Intersect_2;
+
+/*!
+A functor object to construct the circle centered at one point and passing through another one.
+Provides the operator:
+
+`Circle_2 operator()(const Point_2& p, const FT & sr)`,
+
+which returns the circle centered at `p` with `sr` as squared radius.
+*/
+typedef unspecified_type Construct_circle_2;
+
+/*!
+A functor object to compute the point on a geometric primitive which is closest from a query.
+Provides the operator:
+
+`Point_2 operator()(const Primitive::Datum& d, const Point_2& p)`,
+
+which returns the point on `d` that is closest to `p`.
```

Comment:
```suggestion
which returns the point on `d` that is closest to query point `p`.
```
Would this help to make it clear @MaelRL ?

Reply:
See https://github.com/CGAL/cgal/pull/8057#discussion_r1559944470.